docs/help-topics/stonesoup-import/stonesoup-import.yml

```diff
@@ -41,16 +41,21 @@
   content: |-
     We analyze your code, then define and apply deployment settings for each of your application components. Remember, a “component” is an image that we build from source code in a repository. An application is a collection of these components working together.
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too. 
 
-    - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
-    - **Route**: Enter the public URL for your published application.
-    - **Environment variables**: Define your environment variables to customize how we deploy your application. Usually, you define these in a devfile or Dockerfile. If your application doesn’t contain either of these file types, we use an application analyzer to extract the necessary information from your code.
-    - **Runtime**: Did we detect the right runtime? If so, you don’t have to do anything with this field. If you want to change your runtime, click the **Runtime** menu and make a selection.
-    - **Send pull request**: Select this option to create a custom pipeline in your Git repository.
+    - Before you expand all of your configuration options, check the following:
+        - Is the repository correct? Click the GitHub link to check your repository URL or click the repository name above the GitHub link to edit it. When you're satisfied, click the checkmark. To exit editing mode, click the X.
+        - **Runtime**: Did we detect the right one? If so, you don’t have to do anything with this field. If you want to change your runtime, click the runtime name and make a selection.
+    - Expand the section next your repository name, then expand the **Build & deploy configuration** menu to look over these items:
+        - **Dockerfile path**: If we're pointing to the wrong directory for your Dockerfile, click inside this field to make changes.
+        - **Target port**: Is it correct? If not, click in the field to modify it.
+        - **CPU** and **Memory**: Choose how many of each of these you want for your application, depending on your subscription.
```

Comment:
We have no means to charge for RHTAP yet, there is no subscription. The values should be based on what their application's resource requirements are to run successfully (and in many cases, hopefully the default values are safe), but if the cores or memory are smaller than the app needs, it could result in deployment failures or degraded performance once deployed.

Reply:
i see. how does a user know whether the values are too low, other than (after the fact) failure/icky performance? i'm inclined to make a note calling this out, but it's a bad idea if we can't also add helpful (before the fact) guidelines, you know? @mattreid